ingest/validation/validate_metadata.py

```diff
@@ -397,12 +393,16 @@ def collect_jsonschema_errors(metadata, convention, bq_json=None):
     """Evaluate metadata input against metadata convention using JSON schema
     returns False if input convention is invalid JSON schema
     """
-    logger.debug('Begin: collect_jsonschema_errors')
     # this function seems overloaded with its three tasks
     # schema validation, non-ontology errors, ontology info collection
     # the latter two should be done together in the same pass thru the file
     js_errors = defaultdict(list)
     schema = validate_schema(convention, metadata)
+    # truncate jsonfile so appends invoked below are added to an empty file
+    if bq_json:
```

Comment:
I'd move the # truncate comment inside the 'if' block

Reply:
refined comment to indicate method of subsequent appends for clarity
288698f